### ISSUE TITLE
Add IPv6 address for quad100 Magic DNS to avoid Windows timeout behavior

### ISSUE THREAD

**Issue by louislam** [tailscale/tailscale#2168]
As topic.

The resolution time of "short domain" is slow, but the longer one (xxx.xxx.beta.tailscale.net) is fast.
I tested on Windows and Linux. I only experience the issue on Windows. Linux is completely fine.
The issue affects all my apps, such as rdp, ssh, chrome etc.

I wrote a simple php program to test the resolution time.

Below image is my Windows 10 PC, the resolution time is slow on the first query.
![2021-06-18_133842](https://user-images.githubusercontent.com/1336778/122513740-65892000-d03d-11eb-9915-5022f83af667.jpg)

Below image is my Windows Server 2019 on another network. The resolution time is slow everytime.
![2021-06-18_134005](https://user-images.githubusercontent.com/1336778/122514290-4d65d080-d03e-11eb-9747-91a4a3dfc890.jpg)

This is AlmaLinux 8.4 that is running on above Windows Server 2019 (Hyper-V). It is completely fast.
And due to this result, it can clearly says that the issue is not related to my physical location or network.
![2021-06-18_133823](https://user-images.githubusercontent.com/1336778/122514407-8736d700-d03e-11eb-8d51-c832e9c00656.jpg)

This is my simple testing script: test_dns.php
```php
<?php
test("win11");
test("win11");
test("win11");
test("win11");

test("win11.xxx.beta.tailscale.net");
test("win11.xxx.beta.tailscale.net");
test("win11.xxx.beta.tailscale.net");

test("www.google.com");
test("github.com");

function test($domain) {
    $start = microtime(true);
    gethostbyname($domain);
    $time = round((microtime(true) - $start) * 1000, 2);
    echo "[$domain] Resolution Time: $time ms\n";
}
```

Btw, thank you for creating tailscale. I really love this zero-config overlay network!

**Comment by louislam** [comment]
After some investigations, I found out that if I disable IPv6 for all my network adapters, the dns can be solved the domain quickly.

![image](https://user-images.githubusercontent.com/1336778/122652791-42608c80-d173-11eb-8f38-698b67a9c914.png)

**Comment by DentonGentry** [comment]
Rewording this as a feature request: if MagicDNS answered on an IPv6 address I bet it would avoid this issue.

**Comment by DentonGentry** [comment]
As the delay is 2.2 seconds, I believe this is another symptom of https://github.com/tailscale/tailscale/issues/1659 which is fixed in Tailscale 1.30. I'm closing this as fixed, please reopen if 1.30 does not resolve it in your environment.
